engine/src/main/java/org/terasology/rendering/nui/widgets/UIText.java

```diff
@@ -68,27 +68,27 @@
     private TextureRegion cursorTexture;
 
     @LayoutConfig
-    private Binding<String> text = new DefaultBinding<>("");
+    protected Binding<String> text = new DefaultBinding<>("");
 
     @LayoutConfig
-    private boolean multiline;
+    protected boolean multiline;
 
     @LayoutConfig
     private boolean readOnly;
```

Comment:
Consider setting `UIText`'s other private variables and methods (except `logger`, `BLINK_RATE`, `blinkCounter` and `cursorTexture` as they are unlikely to be changed by a child class) to `protected` - don't just make half of class extensible! 😛